repos/system_upgrade/el7toel8/actors/prepareupgradetransaction/actor.py

```diff
@@ -212,7 +212,7 @@ def process(self):
             return
 
         # TODO: Find a better place where to run this (perhaps even gate this behind user prompt/question)
-        preparetransaction.call(['/usr/bin/dnf', 'clean', 'all'])
+        call(['/usr/bin/dnf', 'clean', 'all'])
```

Comment:
split=False, since it is not used, so it's a waste of resources to actually try to split it